app/routes/surveys/components/SurveyEditor/SurveyEditor.js

```diff
@@ -263,10 +266,22 @@ export const initialQuestion = {
   ],
 };
 
+export const hasOptions = (data) => {
+  const message = {}
+    message.questions = []
+    data.questions?.forEach((element, i) => {
+      if (!["multiple_choice","single_choice"].includes(element.questionType)) return
```

Comment:
I would like this to be use an ENUM to avoid bugs later on. Not your fault that the ENUM does not exist, as [this](https://github.com/webkom/lego-webapp/blob/79a65ef786cd43dde0aebbd8c3adfa3f05051dd2/app/routes/surveys/utils.js#L13) utility is not exported.

Would be nice if the type are exported as a constant ENUM from this file, and that we use that here 👍

Reply:
Should I make changes to export `questionStrings` in `utils.js` ?